admin/tests/acceptance/user-list_test.js

```diff
@@ -42,18 +42,33 @@ module('Acceptance | User List', function (hooks) {
       await visit('/users/list');
 
       // then
-      assert.dom('.table-admin tbody tr').exists({ count: nbUser + 1 });
+      assert.dom('.table-admin tbody tr').doesNotExist();
     });
 
     test('it should display the current filter when users are filtered', async function (assert) {
       // given
-      server.createList('user', 12);
+      const nbUser = 1;
+      const result = {
+        data: [
+          {
+            type: 'users',
+            id: '1',
+            attributes: {
+              'first-name': 'Pix',
+              'last-name': 'Aile',
+              email: 'userpix1@example.net',
+            },
+          },
+        ],
+      };
+
+      this.server.get('/users', () => result);
 
       // when
-      await visit('/users/list?email=sav');
+      await visit('/users/list?email=example.net');
 
       // then
-      assert.dom('#email').hasValue('sav');
+      assert.dom('.table-admin tbody tr').exists({ count: nbUser });
```

Comment:
**Remarque**
On perd le sens initial de ce test. 
En lisant le titre, ou voudrait vérifier que les résultats affichés sont bien conformes au filtres appliqués. Sauf qu'ici on ne fait pas d'assertion relative aux filtres, seulement sur le nombre de résultats trouvés.

Un peu plus haut, le test `User detail page can be accessed from user list page` se rapproche déjà un peu plus de l'intention dans son nouvel état, mais les filtres n'apparaissent pas clairement dans le test.

Je propose de n'en garder qu'un seul sur ces deux-là et je mettre davantage l'accent sur les filtres et les résultats de recherche

Reply:
J'ai précisé ma remarque dans mon autre commentaire sur ce fichier